ports/quake.2/Quake 2 - Capture The Flag.sh

```diff
@@ -62,7 +61,7 @@ else
 fi
 
 # Check device type
-if [ "$DEVICE_NAME" = "X55" ] || [ "$DEVICE_NAME" = "RG353P" ] || [ "$DEVICE_NAME" = "RG40XX" ]; then
+if [ "$DEVICE_NAME" = "x55" ] || [ "$DEVICE_NAME" = "RG353P" ] || [ "$DEVICE_NAME" = "RG40XX" ]; then
```

Comment:
There is one other thing here that was actually my mistake.  It's supposed to be RG40XX-H.  Dunno if you'd be so kind to correct this for us while we're at it.
```suggestion
if [ "$DEVICE_NAME" = "x55" ] || [ "$DEVICE_NAME" = "RG353P" ] || [ "$DEVICE_NAME" = "RG40XX-H" ]; then
```